doc/dune-files.rst

```diff
@@ -1540,6 +1540,9 @@ Fields supported in ``<settings>`` are:
 
 - any OCaml flags field. See :ref:`ocaml-flags` for more details.
 
+- ``(link_flags <flags>)`` to specify flags to ocaml when linking an
```

Comment:
```suggestion
- ``(link_flags <flags>)`` to specify flags to OCaml when linking an
```

Reply:
Could you please update this capitalization for OCaml before you merge? We're trying to standardize formatting like this across documentation.